crates/symbolicator-service/src/services/symbolication/js.rs

```diff
@@ -73,12 +73,19 @@ impl SymbolicationActor {
         for raw_stacktrace in &mut raw_stacktraces {
             let num_frames = raw_stacktrace.frames.len();
             let mut symbolicated_frames = Vec::with_capacity(num_frames);
+            let mut prev_frame_token_name = None;
 
             for raw_frame in &mut raw_stacktrace.frames {
-                match symbolicate_js_frame(&mut lookup, raw_frame, &mut missing_sourcescontent)
-                    .await
+                match symbolicate_js_frame(
+                    &mut lookup,
+                    raw_frame,
+                    prev_frame_token_name.clone(),
```

Comment:
```suggestion
                    std::mem::take(&mut prev_frame_token_name),
```
Because:
- in the `Ok` branch, you overwrite it anyway
- in the `Err` branch, you don’t want it to span over more than one (unable to symbolicate) frame anyway